Detect Open Browser fast 5G

Simbiat opened the pull request:
Changed name for `com.wOpenBrowser_12576500` (https://play.google.com/store/apps/details?id=com.wOpenBrowser_12576500), since it does not seem related to `com.wOpenbrowser_13506467` (https://play.google.com/store/apps/details?id=com.wOpenbrowser_13506467) and does have slightly different name on Google Play.

### Description:

Please include a description of this change and which issue it fixes. If no issue exists yet please include context and what problem it solves.

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

Commit:
Update browsers.yml

Changed name for `com.wOpenBrowser_12576500` (https://play.google.com/store/apps/details?id=com.wOpenBrowser_12576500), since it does not seem related to `com.wOpenbrowser_13506467` (https://play.google.com/store/apps/details?id=com.wOpenbrowser_13506467) and does have slightly different name on Google Play.

liviuconcioiu commented:
Hi @Simbiat! You should also add the browser here Parser/Client/Browser.php and a test here Tests/Parser/Client/fixtures/browser.yml

Commit:
Update Browser.php

Added "Open Browser fast 5G" (https://play.google.com/store/apps/details?id=com.wOpenBrowser_12576500)

Commit:
Update browser.yml

Added `Open Browser fast 5G`

Simbiat commented:
Done. Let me know if I need to add/correct something, since this is my first time committing to DD

liviuconcioiu commented:
Everything is okay.
